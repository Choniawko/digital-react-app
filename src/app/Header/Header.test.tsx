import React from "react"
import { shallow, ShallowWrapper } from "enzyme"
import { Header } from "."

describe("Header", () => {
  it("render component", () => {
    const wrapper: ShallowWrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})
