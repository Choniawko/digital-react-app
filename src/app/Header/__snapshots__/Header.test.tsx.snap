// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render component 1`] = `
<header
  className="Header-container"
>
  <img
    alt="logo"
    className="Header-logo-left"
    key="left"
    src="logo.svg"
  />
  <img
    alt="logo"
    className="Header-logo-right"
    key="right"
    src="logo.svg"
  />
</header>
`;
